### DISCUSSION TITLE
EventListener repeat the action many times when using wire:navigate

### DISCUSSION THREAD

**Discussion by enmanuel97** [livewire/livewire#6804] [1. Help]
Hi, I'm new using livewire, I started using it due to the idea of turn my app like an spa, my app use jQuery because the template has it in it's core.

This is my layout `<!DOCTYPE html>
<html lang="{{ app()->getLocale() }}">
	
	<head>
        <title>{{$title ?? ''}} | {{getAppName()}}</title>
        <meta name="csrf-token" content="{{ csrf_token() }}"/>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<meta name="robots" content="noindex,nofollow" />
		<meta name="googlebot" content="noindex,nofollow" />
		<meta name="Googlebot-News" content="noindex,nofollow" />
		<link rel="canonical" href="{{route('login')}}">
		<link rel="shortcut icon" href="{{asset('template/media/images/favicon.png')}}" />
		
		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter:300,400,500,600,700" />
		
		
		@yield('styles')
		
		
		<link data-navigate-once href="{{asset(mix('template/plugins/global/plugins.bundle.css'))}}" rel="stylesheet" type="text/css" />
		<link data-navigate-once href="{{asset(mix('template/plugins/global/plugins-custom.bundle.css'))}}" rel="stylesheet" type="text/css" />
		<link data-navigate-once href="{{asset(mix('template/css/style.bundle.css'))}}" rel="stylesheet" type="text/css" />
		
        @livewireStyles

        @routes
		<script data-navigate-once src="{{asset(mix('template/plugins/global/plugins.bundle.js'))}}"></script>
		<script data-navigate-once src="{{asset(mix('template/js/scripts.bundle.js'))}}"></script>
        <script data-navigate-once src="{{asset(mix('template/plugins/custom/flatpickr/flatpickr.bundle.js'))}}"></script>
        <script data-navigate-once src="{{asset(mix('template/plugins/custom/mask/jquery.mask.bundle.js'))}}"></script>
        <script data-navigate-once src="{{asset(mix('template/plugins/custom/numeric/jquery.alphanum.bundle.js'))}}"></script>
        <script data-navigate-once src="{{asset(mix('template/lang/'.app()->getLocale().'.js'))}}"></script>
        <script data-navigate-once>
            window.lang = window.lang;
            window.userCurrentLanguage = "{{ getLoggedInUser()->language }}";
            window.userData = @json(auth()->user());
            function getRandomColor() {
                const letters = '0123456789ABCDEF';
                let color     = '';
                for (let i = 0; i < 6; i++) {
                    color += letters[Math.floor(Math.random() * 16)];
                }
                return color;
            }
            function getUserImageInitial(data) {
                return `https://ui-avatars.com/api/?name=${data.first_name+' '+data.last_name}&size=100&rounded=true&color=fff&background=${getRandomColor()}`;
            }
        </script>
		<script src="{{ asset(mix('template/modules/custom/dom.js')) }}"></script>
		<script src="{{ asset(mix('template/modules/custom/custom.js')) }}"></script>
		<script src="{{ asset(mix('template/modules/custom/helpers.js')) }}"></script>
		<script src="{{ asset(mix('template/modules/custom/new-edit-modal-form.js')) }}"></script>
		
        @livewireScripts
        <script>
            document.addEventListener('livewire:navigated', () => { 
                KTMenu.init();
                KTScrolltop.init();
                KTScroll.init();
            })
        </script>
	</head>
	
	
	<body id="kt_app_body" data-kt-app-layout="dark-sidebar" data-kt-app-header-fixed="true" data-kt-app-sidebar-enabled="true" data-kt-app-sidebar-fixed="true" data-kt-app-sidebar-hoverable="true" data-kt-app-sidebar-push-header="true" data-kt-app-sidebar-push-toolbar="true" data-kt-app-sidebar-push-footer="true" data-kt-app-toolbar-enabled="true" class="app-default">
		<div class="page-loading d-none">
			<div class="page-loading-inner">
				<div class="page-spinner"></div><span>Cargando...</span>
			</div>
		</div>

		
		<div class="d-flex flex-column flex-root app-root" id="kt_app_root">
			
			<div class="app-page flex-column flex-column-fluid" id="kt_app_page">
				@include('layouts.header', ['breadcrumbs' => $breadcrumbs ?? []])
				
				<div class="app-wrapper flex-column flex-row-fluid" id="kt_app_wrapper">
					@include('layouts.sidebar')
					
					<div class="app-main flex-column flex-row-fluid" id="kt_app_main">
						<div class="d-flex flex-column flex-column-fluid">
							
							<div id="kt_app_content" class="app-content flex-column-fluid">
								
								<div id="kt_app_content_container" class="app-container container-fluid mt-10">
									@yield('content')
								</div>
							</div>
						</div>

						@include('layouts.footer')
					</div>
					
				</div>
				
			</div>
			
		</div>
		

		
		<div id="kt_scrolltop" class="scrolltop" data-kt-scrolltop="true">
			
			<span class="svg-icon">
				<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
					<rect opacity="0.5" x="13" y="6" width="13" height="2" rx="1" transform="rotate(90 13 6)" fill="currentColor" />
					<path d="M12.5657 8.56569L16.75 12.75C17.1642 13.1642 17.8358 13.1642 18.25 12.75C18.6642 12.3358 18.6642 11.6642 18.25 11.25L12.7071 5.70711C12.3166 5.31658 11.6834 5.31658 11.2929 5.70711L5.75 11.25C5.33579 11.6642 5.33579 12.3358 5.75 12.75C6.16421 13.1642 6.83579 13.1642 7.25 12.75L11.4343 8.56569C11.7467 8.25327 12.2533 8.25327 12.5657 8.56569Z" fill="currentColor" />
				</svg>
			</span>
			
		</div>
		
        
        <div class="modal fade" id="globalModal" tabindex="-1" aria-hidden="true">
            
            <div class="modal-dialog"></div>
            
        </div>
		
        @yield('scripts')
	</body>
	
</html>`

As you can see I have a @yield('scripts') and in my module blade file i have `@section('scripts')`
My problem is that I have files for each module and I include it in the section scripts, and all the event listenner that I have in each file repeat many times as i change the url using wire:navigate, I need your help with this problem.

**Comment by SalzBytes** [comment]
Do you mean, you only want to trigger it once?
You should use `livewire:initialized` instead of `livewire:navigated`
see : [doc](https://livewire.laravel.com/docs/javascript#global-livewire-events)
